From a55942a1bdcfa8b7d2365910e50ea9f28fb64ce6 Mon Sep 17 00:00:00 2001
From: Lasse Brun <bruners@gmail.com>
Date: Tue, 15 May 2012 23:01:33 +0200
Subject: [PATCH] Apply patch gsm-1.0.12-memcpy.patch from gentoo

Removes build warning: incompatible implicit declaration of built-in function 'memcpy'
---
 src/code.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/code.c b/src/code.c
index 65dd0a3..c0768db 100644
--- a/src/code.c
+++ b/src/code.c
@@ -8,6 +8,7 @@
 
 #include	"config.h"
 
+#include <string.h>
 
 #ifdef	HAS_STDLIB_H
 #include	<stdlib.h>
-- 
1.7.10.2
